.bazelrc

```diff
@@ -90,6 +92,13 @@ build:linux --action_env=BAZEL_LINKLIBS='-l%:libc++.a -lm -static-libgcc'
 # https://github.com/bazelbuild/bazel/issues/12439#issuecomment-914449079
 build:linux --force_pic
 
+# TODO(later): rustc's -C,codegen-units=1 and ld/lld's -Wl,-O2 on Linux/possibly Windows should
```

Comment:
Nit: I would add the 300k number that you have in the PR description to this comment so that future readers understand it's not a big improvement